packages/design-system-react-native/eslint.config.mjs

```diff
@@ -1,48 +0,0 @@
-import { createConfig } from '@metamask/eslint-config';
```

Comment:
This ESLint config file wasn’t being applied and isn’t supported, so I'm removing it in favor of the root ESLint config.